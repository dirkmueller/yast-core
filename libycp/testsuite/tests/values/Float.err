Parsed:
----------------------------------------------------------------------
300.
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
300.
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.09
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
300.
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.0256
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
3.
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
2.56
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.
----------------------------------------------------------------------
