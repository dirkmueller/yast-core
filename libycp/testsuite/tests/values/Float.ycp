
# ---------------------------------------------------------
#
#  Filename:    Float.ycp
#
#  Purpose:     test cases for YCP test
#
#  Creator:     jd@suse.de
#
#  Maintainer:  jd@suse.de
#
# ---------------------------------------------------------


// this tests includes tests for YCP_Float

# ------- YCP_Float
3e2
3E2
9E-2
3.0e2
2.56E-2
3.0
2.56
0e-2
0.0E2
0e0
#
# a ´normal´ float starting with . like .1 is a YCP_Path !
#
# after the exponent eE there is only ´-´ valid, not ´+´!
#

