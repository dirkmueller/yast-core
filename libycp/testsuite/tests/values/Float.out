(300.)
(300.)
(0.09)
(300.)
(0.0256)
(3.)
(2.56)
(0.)
(0.)
(0.)
